mysql-test/include/connection_net_errors_stats_test.inc

```diff
@@ -0,0 +1,49 @@
+--echo connection_net_errors_stats_test (user= $user, ER_NET_READ_ERROR= $ER_NET_READ_ERROR, ER_NET_READ_INTERRUPTED= $ER_NET_READ_INTERRUPTED)
+
+connect (conn_stat,127.0.0.1,root);
+
+flush status;
+
+# Test ER_NET_READ_ERROR
+if ($ER_NET_READ_ERROR == 1)
+{
+  connect (conn1,127.0.0.1,$user);
+  select "disconnect the connection";
+  dirty_close conn1;
+
+  --source include/wait_until_disconnected.inc
+}
+
+# Test ER_NET_READ_INTERRUPTED
+if ($ER_NET_READ_INTERRUPTED == 1)
+{
+  connect (conn2,127.0.0.1,$user);
+  set session wait_timeout = 1;
+  --real_sleep 2
+  --error 2006, 2013
```

Comment:
Are there mnemonics for 2006 and 2013?

Reply:
Starting with 8.0, client-side errors have mnemonics, starting with CR_.